azext_iot/tests/test_iot_dps_int.py

```diff
@@ -163,12 +163,15 @@ def test_dps_enrollment_x509_lifecycle(self):
 
     def test_dps_enrollment_symmetrickey_lifecycle(self):
         enrollment_id = self.create_random_name('enrollment-for-test', length=48)
+        enrollment_id2 = self.create_random_name('enrollment-for-test', length=48)
         attestation_type = AttestationType.symmetricKey.value
         primary_key = 'x3XNu1HeSw93rmtDXduRUZjhqdGbcqR/zloWYiyPUzw='
         secondary_key = 'PahMnOSBblv9CRn5B765iK35jTvnjDUjYP9hKBZa4Ug='
         device_id = self.create_random_name('device-id-for-test', length=48)
         reprovisionPolicy_reprovisionandresetdata = 'reprovisionandresetdata'
         hub_host_name = '{}.azure-devices.net'.format(hub)
+        webhook_url = 'https://www.test.test'
```

Comment:
Can we use GUID formed URL?

Reply:
Updated

Reply:
This is the test data..

Reply:
Totally understand. My concern is, if during tests, any packets get sent to what is not a real URL for now...but might be a real URL later (concern is if the DPS service sends any ack or other info to a fake webhook url).

If DPS doesn't do anything like that on setting the custom allocation policy then its a moot point.

Reply:
Ok, looks like .test domain can't be registed https://en.wikipedia.org/wiki/.test